packages/cli/package.json

```diff
@@ -9,7 +9,9 @@
     "cli",
```

Comment:
Can we reduce these keywords to 3-5 only. 
lightning design system linter
SLDS Linter
SLDS2 Linter
SLDS Stylelint
SLDS ESLint